diff --git a/score/config_management/config_daemon/code/BUILD b/score/config_management/config_daemon/code/BUILD
index b9a1469..7fdb370 100644
--- a/score/config_management/config_daemon/code/BUILD
+++ b/score/config_management/config_daemon/code/BUILD
@@ -76,6 +76,7 @@ cc_binary(
     deps = [
         "//score/config_management/config_daemon/code/app/details:app",
         "//score/config_management/config_daemon/code/factory/details:mw_factory",
+        "@score_lifecycle_health//score/launch_manager:lifecyclemanager_cc",
         "@score_lifecycle_health//score/launch_manager:runapplication_cc",
     ],
 )
diff --git a/score/config_management/config_daemon/code/app/details/BUILD b/score/config_management/config_daemon/code/app/details/BUILD
index 63e9161..db9b630 100644
--- a/score/config_management/config_daemon/code/app/details/BUILD
+++ b/score/config_management/config_daemon/code/app/details/BUILD
@@ -52,6 +52,7 @@ cc_library(
         "//score/config_management/config_daemon/code/factory:interface_for_unit_test",
         "//score/config_management/config_daemon/code/fault_event_reporter",
         "@score_baselibs//score/scope_exit",
+        "@score_lifecycle_health//score/launch_manager:application_cc",
         "@score_lifecycle_health//score/launch_manager:applicationcontext_mock_cc",
         "@score_logging//score/mw/log",
     ],
diff --git a/score/config_management/config_provider/BUILD b/score/config_management/config_provider/BUILD
index e0d763b..0dd66b9 100644
--- a/score/config_management/config_provider/BUILD
+++ b/score/config_management/config_provider/BUILD
@@ -14,13 +14,6 @@ alias(
     visibility = ["//visibility:public"],
 )
 
-cc_library(
-    name = "config_provider_factory_impl",
-    tags = ["FUSA"],
-    visibility = ["//visibility:public"],
-    deps = ["//score/config_management/config_provider/code/config_provider/factory:factory_socal_r20_11"],
-)
-
 cc_library(
     name = "config_provider_mw_com",
     tags = ["FUSA"],
@@ -28,12 +21,6 @@ cc_library(
     deps = ["//score/config_management/config_provider/code/config_provider/factory:factory_mw_com"],
 )
 
-alias(
-    name = "config_provider_factory_for_unit_test",
-    actual = "//score/config_management/config_provider/code/config_provider/factory:factory_socal_r20_11_for_unit_tests",
-    visibility = ["//visibility:public"],
-)
-
 cc_unit_test_suites_for_host_and_qnx(
     name = "unit_tests",
     test_suites_from_sub_packages = [
@@ -45,14 +32,6 @@ cc_unit_test_suites_for_host_and_qnx(
     visibility = ["//score/config_management:__pkg__"],
 )
 
-test_suite(
-    name = "component_tests",
-    tests = [
-        "//score/config_management/config_provider/test/sct:component_tests",
-    ],
-    visibility = ["//score/config_management:__pkg__"],
-)
-
 clang_tidy_extra_checks(
     name = "clang_tidy_extra_checks",
     extra_features = [
