--- a/BUILD	2026-09-03 18:11:07.439554977 +0300
+++ b/BUILD	2026-09-03 18:11:07.439554977 +0300
@@ -12,7 +12,7 @@
 # *******************************************************************************
 
 load("@score_docs_as_code//:docs.bzl", "docs")
-load("@score_tooling//:defs.bzl", "cli_helper", "copyright_checker", "dash_license_checker", "rust_coverage_report", "setup_starpls", "use_format_targets")
+load("@score_tooling//:defs.bzl", "cli_helper", "copyright_checker", "dash_license_checker", "setup_starpls", "use_format_targets")
 load("//:project_config.bzl", "PROJECT_CONFIG")
 
 # Creates all documentation targets:
@@ -83,21 +83,6 @@
     visibility = ["//visibility:public"],
 )
 
-rust_coverage_report(
-    name = "rust_coverage",
-    bazel_configs = [
-        "per-x86_64-linux",
-        "ferrocene-coverage",
-    ],
-    query = 'kind("rust_test", //src/rust/...)',
-    visibility = ["//visibility:public"],
-)
-
-alias(
-    name = "rust_coverage_report",
-    actual = ":rust_coverage",
-    visibility = ["//visibility:public"],
-)
 
 exports_files(
     ["pyproject.toml"],
